src/GeoTables.jl

```diff
@@ -5,101 +5,21 @@
 module GeoTables
 
 using Meshes
+using Tables
+using Random
 
-import Tables
-import Meshes
+import Base: values
+import Meshes: partition, partsubsets, unview
+import DataAPI: nrow, ncol
+import StatsBase: sample
```

Comment:
Maybe we should move partition and sample to GeoStatsBase.jl and leave the minimum number of methods in this package. Just the ones that are strictly necessary to manipulate the type as a table.